vlsi/sim.mk

```diff
@@ -10,7 +10,7 @@ $(SIM_CONF): $(sim_common_files)
 	echo "  top_module: $(VLSI_TOP)" >> $@
 	echo "  tb_name: ''" >> $@  # don't specify -top
 	echo "  input_files:" >> $@
-	for x in $$(cat $(MODEL_MODS_FILELIST) $(MODEL_BB_MODS_FILELIST) | sort -u) $(MODEL_SMEMS_FILE) $(SIM_FILE_REQS); do \
```

Comment:
I think the some files in `MODEL_BB_MODS_FILELIST` are still needed.
These verilog files will be missing:
```
SimUART.v
SimJTAG.v
SimDRAM.v
SimSerial.v
```
And some `.cc` files will be missing as well (e.g. `mm.cc`, `uart.cc`), but I'm unsure whether they are needed at the moment.

Reply:
Oh I see you've made more changes in previous commit. Let me rerun the whole flow to see if that's still the case.

Reply:
https://github.com/ucb-bar/chipyard/pull/1452/commits/3f80507ce4a2f8ae659da2d27ab81d0a3be8f564#diff-9ac3b0343b35590f2cdaef0c5a042af25c247ba4c7fbb8804ab4a834f96d9215R168-R170 should let the `.v` files in. However, I suspect this will cause those `.cc` files to not make it.

Reply:
I haven't reached post-synthesis simulation yet, but now the synthesis wouldn't work for me since the blackboxed verilog is not found (previously in `.model.bb.f`).
```
Error   : Could not resolve reference. [CDFG-431] [elaborate]
        : The unresolved reference is for instance 'hpu_hpu' in file '/bwrcq/C/allpan/new-chipyard/vlsi/generated-src/chipyard.TestHarness.HPURocketConfig/gen-collateral/RocketTile.sv' on line 931.
        : Use 'set_db hdl_error_on_blackbox false' (in CUI mode) or 'set_attribute hdl_error_on_blackbox false /' (in legacy mode) to cause a warning, rather than an error, when a blackbox is found.
Warning : Ignored attempt to delete blackbox or cell. [CDFG-313]
        : Blackbox 'hpu_BUS_WIDTH256_DIST_ACC_INT_WIDTH13_BUNDLE_INT_WIDTH9_NUM_DIST_REGS8_COMPARE_TYPEx4152474d494e_NUM_BUNDLER_REGS2_SRAM_DEPTH2048'.
```